package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "js-restaurant-page",
+  "version": "1.0.0",
+  "description": "A single page app which shows menu, about and homepage of a restaurant. The app is build using javascript.",
+  "main": "index.js",
+  "scripts": {
```

Comment:
- [ ] [REQUIRED] Please add the necessary run common commands like `build`, `development`
![image](https://user-images.githubusercontent.com/73254040/99142547-a5a5a180-261b-11eb-96f8-24292ee80b76.png)
[Source](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/59622983/)

Reply:
Added build and development scripts in package.json